CHANGELOG.md

```diff
@@ -38,6 +38,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## [v0.50.0-alpha.0](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.50.0-alpha.0) - 2023-06-07
```

Comment:
Adding it here so that we do not have the messy changelog of last release, and we can collapse it at the eind.